Make sure liblqr-1.so's full path is passed to the linker.
--- CMakeLists.txt.orig	2016-06-16 18:45:54 UTC
+++ CMakeLists.txt
@@ -557,6 +557,7 @@ if(DIGIKAM_CAN_BE_COMPILED)
     if(LQR-1_FOUND)
         include_directories(${LQR-1_INCLUDE_DIRS})
         set(liblqr_LIBS ${LQR-1_LIBRARIES})
+        link_directories(${LQR-1_LIBDIR})
     endif()
 
     # Lensfun library rules
